Sources/MongoSwift/ChangeStreamOptions.swift

```diff
@@ -0,0 +1,91 @@
+/// Describes the modes for configuring the fullDocument field of a `ChangeStreamDocument`.
+public enum FullDocument: RawRepresentable, Codable {
+    /// Specifies that the `fullDocument` field will contain a copy of the entire document that was changed from
+    /// some time after the change occurred. If the document was deleted since the updated happened, it will be null.
+    case updateLookup
+    /// For an unknown value. For forwards compatibility, no error will be thrown when an unknown value is provided.
+    case other(String)
+
+    public var rawValue: String {
+        switch self {
+        case .updateLookup:
+            return "updateLookup"
+        case .other(let v):
+            return v
+        }
+    }
+
+    public init?(rawValue: String) {
+        switch rawValue {
+        case "updateLookup":
+            self = .updateLookup
+        default:
+            self = .other(rawValue)
+        }
+    }
+}
+
+/// Options to use when creating a `ChangeStream`.
+public struct ChangeStreamOptions: Codable {
+    /**
+     * Indicates how the `fullDocument` field of a `ChangeStreamDocument` should be filled out by the server.
+     * By default (indicated by a nil value for this option), the `fullDocument` field in the change stream document
+     * will always be present in the case of 'insert' and 'replace' operations (containing the document being inserted)
+     * and will be nil for all other operations.
+     */
+    public let fullDocument: FullDocument?
+
+    /**
+     * A `ChangeStreamToken` that manually specifies the logical starting point for the new change stream.
+     * The change stream will attempt to resume notifications starting after the operation associated with
+     * the provided token.
+     * - Note: A change stream cannot be resumed after an invalidate event (e.g. a collection drop or rename).
+     *         Use the `startAfter` option in those cases instead.
+     * - SeeAlso: https://docs.mongodb.com/manual/changeStreams/#resume-a-change-stream
+     */
+    public let resumeAfter: ChangeStreamToken?
+
+    /// The maximum amount of time in milliseconds for the server to wait on new documents to satisfy a
+    /// change stream query. Uses the server default timeout when omitted.
+    public let maxAwaitTimeMS: Int64?
+
+    /// The number of documents to return per batch. The default is to not send a value.
```

Comment:
nit: I think "the default is to not send a value" is more a drivers internal thing. I would suggest phrasing this like the `maxAwaitTimeMS` suggestion I made above:
"If omitted, the server will use its default batch size."